tracker_package/tracker/src/tracker/tracker.py

```diff
@@ -0,0 +1,122 @@
+import rospy
+import select
+import cv2
+import numpy as np
+import socket
+import sys
+import pickle
+import struct
+import torch
+
+from ltr.data.bounding_box_utils import masks_to_bboxes
+from pytracking.evaluation import Tracker
+from sensor_msgs.msg import Image
+from std_msgs.msg import Float32MultiArray
+from cv_bridge import CvBridge, CvBridgeError
+from tracker.optical_flow import VisualTrackerKLT   
+
+
+class ObjectTracker:
+
```

Comment:
Add some documentation. What makes it different in comparison with the "Tracker" class for instance?
How can one know by just looking at the class without (even minimal) documentation?